### ISSUE TITLE
error on `yarn install`: Argument of type 'FontFace' is not assignable to parameter of type 'global.FontFace'

### ISSUE THREAD

**Issue by oxygen-xx** [MatthewHerbst/react-to-print#432]
```sh
yarn install 
error : 
react-to-print/src/index.tsx(275,68)       
TS2345: Argument of type 'FontFace' is not assignable to parameter of type 'global.FontFace'.
 Type 'FontFace' provides no match for the signature 'new (family: string, source: string | ArrayBuffer, descriptors?: FontFaceDescriptors | undefined): FontFace'.
```

**Comment by MatthewHerbst** [comment]
Hello. Can you please show what versions of `yarn`, `react-to-print`, and `typescript` you are using?

**Comment by oxygen-xx** [comment]
> Hello. Can you please show what versions of `yarn`, `react-to-print`, and `typescript` you are using?

macos 12
Yarn, 1.22.17
react-to-print , 2.14.0
typescript , 4.5.2

**Comment by MatthewHerbst** [comment]
Thanks for the information. I'm not able to replicate the error. Do you have anything else configured? Could you maybe share your `package.json` and `tsconfig.json` files please?

![Screen Shot 2021-11-18 at 10 43 38 AM](https://user-images.githubusercontent.com/1287375/142477318-6849d8d7-2f92-4e78-82a4-2f1a3e0a87d4.png)

**Comment by oxygen-xx** [comment]
I just cloned this project, and then yarn install 

git clone https://github.com/gregnb/react-to-print.git
yarn install

**Comment by MatthewHerbst** [comment]
This seems to be something to do with a breaking change in TypeScript, since it works with earlier versions. Should have a fix out shortly, thank you

**Comment by MatthewHerbst** [comment]
The breaking change occurred specifically in TypeScript v4.4. It seems TypeScript added `FontFace` to it's lib files, so we seemingly don't need to define the interface anymore! https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/1029#issuecomment-869224737 Working on the fix now

**Comment by MatthewHerbst** [comment]
The fix for this has been published in v2.14.1, please let me know if you run into any other issues. Thank you!

**Comment by oxygen-xx** [comment]
> The fix for this has been published in v2.14.1, please let me know if you run into any other issues. Thank you!

thank you , 2.14.1 is fine!.
